### PR TITLE
adding basic tests

### PR DESCRIPTION
**Pull request by khamui**

### CODE REVIEW

**Comment by fuzzylogic2000** [comment]
The test is failing...

**Comment by goapunk** [comment]
@khamui @fuzzylogic2000 I added a commit which tests the successful login

File: containers/Auth/__tests__/LoginScreen.test.js [hunk 1]
```diff
@@ -11,27 +13,31 @@ test('Test LoginScreen Snapshot', () => {
...
 test('Test Login, no credentials', async () => {
   render(<LoginScreen />);
-  const submitButton = screen.getByTestId('submit-button');
+  const submitButton = screen.getByRole('button');
   fireEvent(submitButton, 'press');
   const validationErrors = await screen.findAllByText(/Please enter your password/);
   expect(validationErrors).toBeTruthy();
 });
 
 // add TestIDs to the inputs in LoginScreen.js if you want to unskip
```
Comment (khamui) [inline containers/Auth/__tests__/LoginScreen.test.js:22]: think you can remove this comment now

**Review by khamui — COMMENTED** [review]
really nice! just the comment can go and then feel free to merge!

**Review by goapunk — APPROVED** [review]
cool!

**Comment by khamui** [comment]
very nice!

**Pull request merged**